feature-tests/tests/resources/step-definitions/invoke-multiple-users-accounts.step.ts

```diff
@@ -0,0 +1,76 @@
+import { defineFeature, loadFeature } from 'jest-cucumber';
+import { generateRandomTestUserId } from '../../../utils/generate-random-test-user-id';
+import { sendSQSEvent } from '../../../utils/send-sqs-message';
+import { invokeGetAccountState } from '../../../utils/invoke-apigateway-lambda';
+import { updateItemInTable } from '../../../utils/dynamo-database-methods';
+import { InformationFromTable } from '../../../utils/utility';
+import * as fs from 'node:fs';
+
+const feature = loadFeature('./tests/resources/features/aisGET/InvokeMultipleUsers-HappyPath.feature');
+
+defineFeature(feature, (test) => {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  let response: any;
+  const listOfUsers: string[] = [];
+
+  test('Happy Path - create multiple users - Returns Expected Data for <aisEventType>', ({
+    given,
+    and,
+    when,
+    then,
+  }) => {
+    given(
+      /^I invoke an API to retrieve the (.*) status to the (.*) accounts. With history (.*)$/,
+      async (aisEventType, numberOfUsers, historyValue) => {
+        for (let index = 0; index < numberOfUsers; index++) {
+          const testUserId = generateRandomTestUserId();
+          await sendSQSEvent(testUserId, aisEventType);
+          response = await invokeGetAccountState(testUserId, historyValue);
+          expect(response.intervention.description).toBe('AIS_ACCOUNT_SUSPENDED');
+          listOfUsers.push(testUserId);
+        }
+        await Promise.allSettled(listOfUsers);
+      },
+    );
+
+    and(/^I update the Id reset flag to TRUE$/, async () => {
+      const updateResetPasswordItemInTable: InformationFromTable = {
+        updatedAt: response.intervention.updatedAt,
+        sentAt: response.intervention.sentAt,
+        appliedAt: response.intervention.appliedAt,
+        intervention: response.intervention.description,
+        suspended: response.state.suspended,
+        reproveIdentity: response.state.reproveIdentity,
+        resetPassword: true,
+        blocked: response.state.blocked,
+      };
+      for (const listOfUser of listOfUsers) {
```

Comment:
minor comment here but could we get 
```suggestion
      for (const user of listOfUsers) {
```
just to make it a bit more readable?

Reply:
updated to user